addon/components/infinity-loader.js

```diff
@@ -13,6 +13,7 @@ export default Ember.Component.extend({
   destroyOnInfinity: false,
   developmentMode: false,
   scrollable: null,
+  didSendActionFromScrollEvent: false,
```

Comment:
This state is really just tracking when the infinityModel is actually loading - so we should make the route the canonical controller for this state.

That's how we communicate to the infinity-loader that it has loaded all the records, so I feel like we should keep that uniform.